### DISCUSSION TITLE
while upgrading keycloak from version 23 to 25, facing: Failed authentication: java.lang.RuntimeException: java.lang.ClassNotFoundException: Provider for jakarta.ws.rs.client.ClientBuilder cannot be found

### DISCUSSION THREAD

**Discussion by AdityaMhetras** [quarkusio/quarkus#42669] [Community]
Aug 21 07:34:36 xxxx d01kc[2326]: 2024-08-21 07:34:36,129 WARN  [org.keycloak.services] (executor-thread-2) KC-SERVICES0013: Failed authentication: java.lang.RuntimeException: java.lang.ClassNotFoundException: Provider for jakarta.ws.rs.client.ClientBuilder cannot be found
Aug 21 07:34:36 xx d01kc[2326]: #011at jakarta.ws.rs.client.ClientBuilder.newBuilder(ClientBuilder.java:75)
Aug 21 07:34:36 xx d01kc[2326]: #011at jakarta.ws.rs.client.ClientBuilder.newClient(ClientBuilder.java:86)
Aug 21 07:34:36 xx d01kc[2326]: #011at xx.xx..updateInfo(xxx.java:273)

**Comment by sberyozkin** [comment]
@AdityaMhetras Is it logged at the Keycloak level ? Please report it in the Keycloak project if yes.

**Reply by AdityaMhetras** [reply]
Sure, I will report in the keycloak project
